Dockerfile

```diff
@@ -1,17 +1,12 @@
-FROM centos:centos7
+FROM openshift/origin-release:golang-1.11 as builder
 
 # Add everything
 ADD . /usr/src/sriov-cni
 
-ENV INSTALL_PKGS "git golang"
-RUN yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    cd /usr/src/sriov-cni && \
-    ./build && \
-    yum autoremove -y $INSTALL_PKGS && \
-    yum clean all && \
-    rm -rf /tmp/*
+RUN cd /usr/src/sriov-cni && make build
 
+FROM openshift/origin-base
+COPY --from=builder /usr/src/sriov-cni/build/sriov /usr/bin/
```

Comment:
This step is not needed. The entrypoint copies the binary to /opt/cni/bin directory anyway. We don't expect the binary to be run within a container.

Reply:
@ahalim-intel perhaps location may be different but since it's a multi-stage build, I think we need to copy it into the runtime container don't we?

Reply:
Yes, you're absolutely right! Totally didn't think of multi-staged build :)